tests/components/myuplink/conftest.py

```diff
@@ -1,59 +1,102 @@
-"""Common fixtures and mocks for myplink tests."""
-
+"""Test helpers for myuplink."""
 from collections.abc import Generator
-from unittest.mock import MagicMock, patch
+import time
+from unittest.mock import AsyncMock, MagicMock, patch
 
 import pytest
 
+from homeassistant.components.application_credentials import (
+    ClientCredential,
+    async_import_client_credential,
+)
 from homeassistant.components.myuplink.const import DOMAIN
 from homeassistant.core import HomeAssistant
 from homeassistant.setup import async_setup_component
 
-from tests.common import MockConfigEntry, load_json_value_fixture
+from .const import CLIENT_ID, CLIENT_SECRET
+
+from tests.common import MockConfigEntry, load_fixture, load_json_value_fixture
+
+
+@pytest.fixture(name="jwt")
+def load_jwt_fixture():
+    """Load Fixture data."""
+    return load_fixture("jwt", DOMAIN)
```

Comment:
Is it important to have this mock token in jwt format? Maybe just set a fake value in the mock config entry data?

Reply:
Nope, took all jwt stuff away.